article/sections/_9_conclusion.tex

```diff
@@ -0,0 +1,42 @@
+\section{Conclusion}
+\label{sec:conclusion}
+
+Travel demand problems aim at forecasting the impact of proposed project and policies. These problems are causal in nature. 
+To our surprise, there has been very little mention of causal inference in the field of travel demand modeling.
```

Comment:
Overall, given that this is a conclusion, can we be as broad as possible by relating our work not just to transportation and travel demand modeling but to choice modeling overall.

Also, I don't know if the slow adoption is surprising as much as sad. I'm suggesting removing very as well simply because its the literary equivalent of typing in all caps.
```suggestion
To date, choice modelers have not routinized the use of causal inference in the field of travel demand modeling.
```